src/main/java/com/netgrif/application/engine/pdf/generator/config/types/PdfBooleanFormat.java

```diff
@@ -2,24 +2,26 @@
 
 import lombok.Getter;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
 
 public enum PdfBooleanFormat {
-
-    SINGLE_BOX_SK(Collections.singletonList("")),
-    SINGLE_BOX_WITH_TEXT_SK(Collections.singletonList("Áno")),
-    DOUBLE_BOX_WITH_TEXT_SK(Arrays.asList("Áno", "Nie")),
-    SINGLE_BOX_EN(Collections.singletonList("")),
-    SINGLE_BOX_WITH_TEXT_EN(Collections.singletonList("Yes")),
-    DOUBLE_BOX_WITH_TEXT_EN(Arrays.asList("Yes", "No")),
+    BOX_WITH_TEXT_SK(new Locale("SK"), Map.of(Boolean.TRUE, "Áno", Boolean.FALSE, "Nie")),
```

Comment:
In the future it would be better to have global config for i18n, maybe using property files like in this article:https://www.baeldung.com/spring-boot-internationalization